### PR TITLE
Remove support for request accessible format form pilot

### PR DESCRIPTION
**Pull request by KludgeKML**
Now that there are no longer any departments taking part, remove code 
while we rethink.

https://trello.com/c/iZEzpeFy/1340-remove-dfe-from-the-accessible-format-pilot

⚠️ This repo is Continuously Deployed: make sure you [follow the guidance](https://docs.publishing.service.gov.uk/manual/development-pipeline.html#merge-your-own-pull-request) ⚠️

### CODE REVIEW

**Comment by KludgeKML** [comment]
Ah, nice spot. Also, I definitely wasn't aware of the other two apps to adjust, I'll get on that now. Thanks for the heads up!

File: test/unit/presenters/publishing_api/publication_presenter_test.rb [hunk 1]
```diff
@@ -154,27 +154,6 @@ def present(edition)
...
 
-  test "documents include the alternative format contact email with a direct email link" do
-    publication = create(:publication, :with_command_paper)
-    GovukPublishingComponents::Presenters::AttachmentHelper.stub_const(:EMAILS_IN_ACCESSIBLE_FORMAT_REQUEST_PILOT, []) do
-      presented_item = present(publication)
-      document = presented_item.content[:details][:documents].first
-      assert document.include?("This file may not be suitable for users of assistive technology.")
-      assert document.include?("mailto:#{publication.alternative_format_provider.alternative_format_contact_email}")
-    end
-  end
```
Comment (1pretz1) [inline test/unit/presenters/publishing_api/publication_presenter_test.rb:165]: We probably want to keep the premise of this test, probably just a copy / paste from the previous version: https://github.com/alphagov/whitehall/blame/86bbd48cfc106ff05960b407fa04f6d7bb234338/test/unit/presenters/publishing_api/publication_presenter_test.rb#L153-L159

Comment (KludgeKML) [reply test/unit/presenters/publishing_api/publication_presenter_test.rb:165]: Ah yes, I think I got confused looking at the original PR. Thanks! Reinstated.

Comment (1pretz1) [reply test/unit/presenters/publishing_api/publication_presenter_test.rb:165]: Awesome, cheers!

**Review by 1pretz1 — APPROVED** [review]
Changes look good @KludgeKML 👍 The e2e tests are now failing though. I believe the errors are not due to this change though and relate to some other Whitehall work:(https://github.com/alphagov/publishing-e2e-tests/pull/489/files & https://github.com/alphagov/whitehall/pull/6632). Both PR's are merged / deployed. Thoughts on these errors @hannako?

**Comment by KludgeKML** [comment]
Let me rebase on main, if the changes are merged that should help.

**Comment by KludgeKML** [comment]
Looks like it did. :)

**Pull request merged**